packages/atomic/src/components/commerce/atomic-product/atomic-product.pcss

```diff
@@ -39,11 +39,17 @@
       }
 
       @screen mobile-only {
-        &.image-large,
+        &.image-large {
+          atomic-product-section-children .product-child {
+            @mixin aspect-ratio-h 1 / 1, auto;
+            width: 16.65%;
+          }
+        }
         &.image-small {
           atomic-product-section-children .product-child {
             @mixin aspect-ratio-h 1 / 1, auto;
             width: 16.65%;
+            max-width: 4.75rem;
           }
```

Comment:
This prevents the product children images from getting too big relative to the main image on wide mobile screen sizes.

Before:
![image](https://github.com/user-attachments/assets/b1a50dca-2be1-4d9b-bb85-17e5fee84fb7)

After:
![image](https://github.com/user-attachments/assets/756b1aa6-9406-4423-9253-6f53a967ef94)